TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -28,6 +28,10 @@ public class TypeCobolConfiguration
         public ExecutionStep ExecToStep = ExecutionStep.Generate; //Default value is Generate
         public string ErrorFile = null;
         public string skeletonPath = "";
+        public string LogFile = null;
+
+        //Log file name
+        public const string LogFileName = "TypeCobol.CLI.log";
```

Comment:
I'd rather name this const `DefaultLogFileName` so we can clearly understand its purpose.

Reply:
Agreed.